content/guides/concepts/dnslink.md

```diff
@@ -7,70 +7,57 @@ menu:
 
 **NOTE: The info below is a quick guide to DNSLink. For a more complete guide, including tutorials, usage examples and FAQs, check out [dnslink.io](http://dnslink.io/).**
 
-## About DNSLink
+## What is DNSLink?
 
-DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map
-a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your
-DNS records, you can use them to always point to the latest version of an
-object in IPFS (remember that an IPFS object’s address changes if you modify
-the object). Because DNSLink uses DNS records, the names it produces are also
-usually easy to type and read.
+DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your DNS records, you can use them to always point to the latest version of an object in IPFS (remember that an IPFS object’s address changes if you modify the object). Because DNSLink uses DNS records, the names it produces are also usually easy to type and read.
 
-A DNSLink address looks like an [IPNS](/guides/concepts/ipns) address, but it
-uses a domain name in place of a hashed public key:
+A DNSLink address looks like an [IPNS](/guides/concepts/ipns) address, but it uses a domain name in place of a hashed public key:
 
 ```
-/ipns/ipfs.io
+/ipns/myexampledomain.org
 ```
 
 Just like normal IPFS addresses, they can include links to other files:
```

Comment:
> Just like normal IPFS addresses, they can include links to other files:

This is supposed to be an intro to the concept of DNSLink but I think it's still worth clarifying that DNSLink addresses can link to other types of resources (e.g. directories and symlinks) that IPFS/UnivFS supports, and **not just files.**

Reply:
Good call - incorporating.